### PR TITLE
Proofread docs

### PR DESCRIPTION
**Pull request by greg0ire**
Best reviewed commit by commit. I solemnly swear the first one is solely about wrapping lines.

### CODE REVIEW

File: docs/en/annotations.rst [hunk 1]
```diff
@@ -67,26 +88,33 @@ To anticipate the configuration section, making the above PHP class work with Do
...
+``AnnotationReader`` instance:
 
 .. code-block:: php
 
     $reader = new \Doctrine\Common\Annotations\AnnotationReader();
 
-This creates a simple annotation reader with no caching other than in memory (in php arrays).
-Since parsing docblocks can be expensive you should cache this process by using
-a caching reader.
+This creates a simple annotation reader with no caching other than in
```
[the remark this answers is not in the document]
Comment (greg0ire) [reply docs/en/annotations.rst]: Yeah it's a bit confusing, I will leave this one as is.

File: docs/en/annotations.rst [hunk 2]
```diff
@@ -257,7 +257,7 @@ PHP Imports
 
 By default the annotation reader parses the use-statement of a php file
 to gain access to the import rules and register them for the annotation
-processing. Only if you are using PHP Imports you can validate the
+processing. Only if you are using PHP Imports can you validate the
```
Comment (stof) [inline docs/en/annotations.rst]: This change looks weird to me. `can you validate` looks like a question to me, but this is not a question.

File: docs/en/custom.rst [hunk 3]
```diff
@@ -20,7 +20,7 @@ Inject annotation values
 ------------------------
 
 The annotation parser checks if the annotation constructor has arguments,
-if so then we will pass the value array, otherwise will try to inject
+if so then we will pass the value array, otherwise we will try to inject
```
Comment (stof) [inline docs/en/custom.rst]: shouldn't we use `it` rather than `we` to refer to the parser ?

File: docs/en/annotations.rst (hunk 2 above)
Comment (alcaeus) [reply docs/en/annotations.rst]: This is fine from a grammar perspective 👍

File: docs/en/custom.rst (hunk 3 above)
Comment (alcaeus) [reply docs/en/custom.rst]: Agree. @greg0ire can you make that change please?

Comment (greg0ire) [reply docs/en/custom.rst]: Oh right I forgot that PR, let me change this!

**Comment by alcaeus** [comment]
Thanks @greg0ire!

**Pull request merged**